public/main.js

```diff
@@ -45,4 +55,85 @@ function sendData(data) {
   return result
 }
 
+//checking email against return array for existing user
+function emailMatches(array) {
+  var input = document.getElementById('email-input')
+  return array.filter(function(person) {
+    return person.email === input.value
+  })
+}
+
+function showUser(user) {
+  var inputs = document.querySelector('.account')
```

Comment:
You should refactor this function to rebuild the user element from scratch and swap it into the DOM. It relies too much on the current structure of the document.